### ISSUE TITLE
ASR Autosplitters from XML not set up correctly on old splits

### ISSUE THREAD

**Issue by PARTYMANX** [LiveSplit/LiveSplit#2384]
Under certain conditions, Autosplitting Runtime autosplitters from the autosplitter XML aren't being set up correctly (the path to the script isn't being set).  I'd just received a report of the autosplitter not working when pulled through the XML automatically and was able to replicated it immediately.

Context: We both were trying to use the THPS autosplitter (specifically for THPS3) automatically set up through the "edit splits" menu.  When I clicked "activate", I noticed that in Livesplit logging, there was nothing coming from the autosplitter.  After some looking around, I noticed that in the settings dialog, there was no path set for the script.  After setting the path manually, it worked as expected.  

I suspect this may be due to both of us having splits dating from before the new ASR splitter was added to the XML (merged Oct 4th of 2023, my splits date back to 2022).  On another PC, I set it up without issue (this PC did not have Livesplit set up previously), and I was also successful in setting up the autosplitter from a blank set of splits.

Here are my old splits files and layout to assist in tracking down the bug:
[oldthpssplits.zip](https://github.com/LiveSplit/LiveSplit/files/13803738/oldthpssplits.zip)

**Comment by ImAciidz** [comment]
Can confirm I also get this bug. Recently the Left 4 Dead 1/2 autosplitter was rewritten with the autosplitting runtime & I've been testing it by editing my autosplitters xml to point to the wasm file instead of the asl (I set the xml to read only...yes I'm aware I could just use it through my layout but whatever lol), but when I open splits that I haven't used since switching to the runtime, it points to the path of the old asl file (or in the case of splits with no stored autosplitter path/settings, the path stays blank). I would've reported this sooner but I originally assumed the bug was caused by me doing this bs with the autosplitters xml and/or the fact that the wasm file doesn't exist on the internet (yet).

In case it's of any concern, here's what I replaced the existing L4D entries in the xml with:
```xml
  <AutoSplitter>
    <Games>
      <Game>Left 4 Dead 2</Game>
      <Game>Left 4 Dead 2 Category Extensions</Game>
      <Game>Left 4 Dead 2 Custom Campaigns</Game>
      <Game>Left 4 Dead</Game>
      <Game>Left 4 Dead Category Extensions</Game>
    </Games>
    <URLs>
      <URL>https://raw.githubusercontent.com/SirWillian/Autosplitters/master/L4D2/L4D.wasm</URL>
    </URLs>
    <Type>Script</Type>
    <ScriptType>AutoSplittingRuntime</ScriptType>
    <Description>Load Removal and Auto Splitting are available. (By bill_play3, 2838, and aciidz)</Description>
    <Website>https://github.com/SirWillian/Autosplitters/tree/l4d-asr</Website>
  </AutoSplitter>
```

Attached below are some splits affected by the bug (No Mercy has old autosplitter settings, Passing is blank as I mainly used the asl through my layout in the past)

[splits.zip](https://github.com/LiveSplit/LiveSplit/files/13855618/splits.zip)
